src/main/resources/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate/config.jelly

```diff
@@ -47,6 +47,9 @@
     <f:textbox default="1"/>
   </f:entry>
   <f:advanced>
+    <f:entry title="${%Dns Search Domains}" field="dnsSearchDomains">
```

Comment:
Suggestion: DNS is a common acronym and should be written in capital letters when enduser facing